packages-native/util-widgets/src/test.ts

```diff
@@ -1,7 +1,10 @@
 import { ActionValue, DynamicValue, ValueStatus } from "mendix";
 
-export function dynamicValue<T>(value?: T): DynamicValue<T> {
-    return value == null ? { status: ValueStatus.Loading, value: undefined } : { status: ValueStatus.Available, value };
+export function dynamicValue<T>(loading: boolean, value?: T): DynamicValue<T> {
```

Comment:
loading should be optional, then you dont need to change all the methods that are using this one. Use `Option<T>` instead then you can define value before and loading can be optional.

Reply:
`declare type Option<T> = T | undefined;`